src/main/java/se/sundsvall/parkingpermit/Constants.java

```diff
@@ -33,6 +33,8 @@ private Constants() {}
 	public static final VariableValueDto TRUE = new VariableValueDto().type(ValueType.BOOLEAN.getName()).value(true);
 	public static final VariableValueDto FALSE = new VariableValueDto().type(ValueType.BOOLEAN.getName()).value(false);
 
+	//TODO Remove when API is updated
```

Comment:
Maybe be a bit more specific? Are we waiting for a story to be completed? 

I.e: what are supposed to be updated in order for this variable to be removed?